scripts/linters/pre_commit_linter.py

```diff
@@ -453,6 +454,17 @@ def _print_errors_stacktrace(errors_stacktrace):
         ' above errors gets fixed')
 
 
+def _get_space_separated_linter_name(linter):
+    """Returns the space separated name of the linter class.
+
+    Returns:
+        str. Space separated name of the linter class.
+    """
+    return re.sub(
+        r'((?<=[a-z])[A-Z]|(?<!\A)[A-Z](?=[a-z]))',
```

Comment:
I was thinking of having name property for each linter class but considering that we are going to have the name for each function very soon, I'm fine with this change!